components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,765 @@
+$$ (def-h)
+
+The letter $h$ is used because this is the familiar variable for thickness in Boussinesq primitive equations. In the Boussinesq case the thickness equation describes conservation of volume because $h_k^{Bouss}$ is volume normalized by horizontal area, resulting in a height,
+
+$$
+h_k^{Bouss}(x,y,t)
+ \equiv \int_{z_k^{bot}}^{z_k^{top}} dz.
+$$ (def-h-bouss)
+
+In this document we remain with the more general non-Boussinesq case, where $h_k$ is mass per unit area (kg/m$^2$). Since horizontal cell area remains constant, the thickness equation is a statement of conservation of mass.
+
+Throughout this derivation we can write all equations equivalently in $z$-coordinates (depth), or in $p$-coordinates (pressure). From the hydrostatic equation, any quantity $\varphi$ may be integrated in $z$ or $p$ as
+
+$$
+\int_{z_k^{bot}}^{z_k^{top}} \varphi \rho dz
+= \frac{1}{g} \int_{p_k^{top}}^{p_k^{bot}} \varphi dp.
+$$(depth-pressure-integral-conversion)
+
+We can convert from an interfacial depth surface $z^{top}$ to a pressure surface $p^{top}$ with the hydrostatic equation [](continuous-hydrostatic-pressure-final):
+
+$$
+p^{top}(x,y) =  p^{surf}(x,y) + \int_{z^{top}(x,y)}^{z^{surf}(x,y)} \rho g dz
+$$ (def-p-surf)
+
+where $z^{surf}$ is the sea surface height and $p^{surf}$ is the surface pressure at $z^{surf}$ imposed by the atmosphere or floating ice. Note that pressure increases with depth.  This means that positive $p$ points downward, so that the $top$ and $bot$ extents of the integration limits are flipped in [](#depth-pressure-integral-conversion).
+
+For any three-dimensional quantity $\varphi(x,y,z,t)$, the mass-thickness-averaged quantity in layer $k$ is defined as
+
+$$
+\varphi_k(x,y,t)
+\equiv \frac{\int_{z_k^{bot}}^{z_k^{top}} \rho \varphi dz}{\int_{z_k^{bot}}^{z_k^{top}} \rho dz}
+= \frac{\int_{z_k^{bot}}^{z_k^{top}} \rho \varphi dz}{h_k}
+$$(def-mass-thickness-average)
+
+At this point our derivation has not made any assumptions about density, and may be used for both Boussinesq and non-Boussinesq fluids. A Boussinesq derivation would now assume small variations in density and replace $\rho(x,y,z,t)$ with a constant $\rho_0$ everywhere but the pressure gradient coefficient. In that case $\rho$ divides out in [](#def-mass-thickness-average) the Boussinesq layer quantities would simply be thickness-weighted averages.
+
+We can now derive the layered equations. Integrate the continuous equations [](continuous-momentum-final), [](continuous-mass-final), [](continuous-tracer-final) in $z$ from $z_k^{bot}$ to $z_k^{top}$,
+
+$$
+\frac{\partial }{\partial t} \int_{z_k^{bot}}^{z_k^{top}} {\bf u} dz
++  \int_{z_k^{bot}}^{z_k^{top}}  \left( \zeta + f \right){\bf u}^\perp dz
++  \int_{z_k^{bot}}^{z_k^{top}} \frac{\partial }{\partial z} \left(  w{\bf u} \right) dz
+  =  \int_{z_k^{bot}}^{z_k^{top}}  \left[- \frac{1}{\rho}\nabla_z p - \nabla_z K
++ {\bf D}^u + {\bf F}^u \right] dz
+$$ (z-integration-momentum)
```

Comment:
To get to the layered equations, each of these needs to be an integral over `\rho dz`.  This means, we need to start with the momentum, rather than the velocity equation here.